src/mainframe/models/schemas.py

```diff
@@ -16,6 +19,57 @@ class Error(BaseModel):
     detail: str
 
 
+class Package(BaseModel):
+    """Model representing a package queried from the database."""
+
+    scan_id: str
+    name: str
+    version: str = ""
+    status: str = ""
+    score: Optional[int] = 0  # Just for the sake of texting, remove upon merge
```

Comment:
`score` should always be returned (even if it's 0)
```suggestion
    score: int
```